bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gpi/Gpi.java

```diff
@@ -792,6 +818,10 @@ public static Apodizer valueOf(String name, Apodizer value) {
             }
             return None.instance();
         }
+
+        public static Apodizer[] validValues() {
+            return Arrays.stream(values()).filter(a -> !a.isObsolete()).toArray(Apodizer[]::new);
+        }
```

Comment:
Just out of curiosity... why `Apodizer.validValues`, but `ObservingMode.nonObsoleteValues`?
They seem to be very similar so I was just wondering about the name difference.